### DISCUSSION TITLE
Profile level

### DISCUSSION THREAD

**Discussion by Eze-Din** [community/community#33584] [Other Feature Feedback, Questions, & Ideas] [answered]
When does my github profile level, like achievement, displayed on my profile?

**Comment by cosmicqbit** [comment, marked as answer]
Awesome man! 😎 You have unlocked the Pull Shark badge / achievement.

**Reply by Eze-Din** [reply]
are you serious?

**Reply by cosmicqbit** [reply]
Of course
<img width="359" alt="image" src="https://user-images.githubusercontent.com/75565639/191460675-cf79ed0a-48cd-48e0-8509-671ddcf8bd99.png">

**Reply by Eze-Din** [reply]
I see thanks!

**Comment by zaahiral** [comment]
should be almost instant, talking from experience, however if you make the change in a private repo you must enable the setting to show contributions in private repositories in settings

**Reply by Eze-Din** [reply]
So why it needs to be private, if others can contribute?

**Reply by cosmicqbit** [reply]
Others can't contribute to your private repos, when he said "show contributions in private repos", he meant those green square dots on contribution graph.
